ingestion/src/metadata/cli/db_dump.py

```diff
@@ -64,6 +67,40 @@ def dump_all(tables: List[str], engine: Engine, output: Path) -> None:
                 file.write(insert)
 
 
+def dump_entity_custom(engine: Engine, output: Path, inspector) -> None:
+    """
+    This function is used to dump entities with custom handling
+    """
+    with open(output, "a") as file:
+        for table, data in EXCLUDED_TABLES.items():
```

Comment:
Instead of EXCLUDED_TABLES might we name it `custom_tables` or something? to understand a bit better how this is being used

Reply:
Yes, Sounds good, Will do that.